README.md

```diff
@@ -1,6 +1,8 @@
 # Determinator
 
-A gem that works with _Florence_ to deterministically calculate whether an **actor** should have a feature flag turned on or off, or which variant they should see in an experiment.
+A gem that works with [actor-tracking](https://github.com/deliveroo/actor-tracking) to deterministically calculate whether an **actor** should have a feature flag turned on or off, or which variant they should see in an experiment.
```

Comment:
I think we need to change the English here; Florence's UI is _currently_ hosted within Actor Tracking, but this won't be the case for ever, and the phrasing below will make sure people can find the right repo, but also know the terminology that makes sense given other documentation out there. What do you think?

```markdown
A gem that works with [Florence](https://github.com/deliveroo/actor-tracking)
```

Reply:
What about making it that explicit in the README itself?

>A gem that works with Florence to deterministically calculate whether an **actor** should have a feature flag turned on or off, or which variant they should see in an experiment. Florence's UI is *currently* hosted within [actor-tracking](https://github.com/deliveroo/actor-tracking).

Reply:
In other words, Florence is a separate system, and determinator is a client for Florence? This seems a bit different from what it says in `background.md` (below), i.e. that Florence is a _suite of tools_ and Determinator is part of that suite.

@jphastings is Determinator part of Florence or is it a client for Florence? Sorry for the nitpicking, but I think being clear on this will help the next people who come along (like us) and try to learn about it from scratch 😄

Reply:
I think the answer to your question @igelstorm might be "yes" 😳 Determinator is part of the Florence (suite), gets the details it needs from Florence (the service) which are updated using Florence (UI).

I think @picandocodigo's phrasing works really well because its explicit about the relationships between the components ("A gem that works with Florence" — where "Florence" here would work equally well being the UI or the server, depending on the reader's comfort with back-end systems design; and "Florence's UI is … hosted" —  being clear that if you want the UI then go here)

Is that satisfying enough?